internal/api/v1beta1/authenticate.go

```diff
@@ -87,9 +88,36 @@ func (h Handler) Authenticate(ctx context.Context, request *frontierv1beta1.Auth
 		}
 	}
 
+	if request.GetStrategyName() == "passkey" {
```

Comment:
use const here instead of string for comparison

Reply:
done